src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/search_preferences.yml

```diff
@@ -22,5 +22,43 @@ admin_alias_search_search_term_action:
   path: /
   methods: [ GET, POST ]
   defaults:
-    _controller: PrestaShopBundle\Controller\Admin\Configure\ShopParameters\SearchController::searchAliasesForAssociationAction
+    _controller: PrestaShopBundle\Controller\Admin\Configure\ShopParameters\SearchAliasController::searchAliasesForAssociationAction
     _legacy_controller: AdminSearchConf
+
+admin_search_alias_delete:
```

Comment:
Please remember to put the legacy_link at some point, or the CI will block the PR